ydb/library/actors/wilson/wilson_uploader.cpp

```diff
@@ -179,15 +270,20 @@ namespace NWilson {
             template<typename T>
             void ScheduleWakeup(T&& deadline) {
                 if (!WakeupScheduled) {
-                    TActivationContext::Schedule(deadline, new IEventHandle(TEvents::TSystem::Wakeup, 0, SelfId(), {},
-                        nullptr, 0));
+                    TActivationContext::Schedule(deadline,
+                                                 new IEventHandle(SelfId(), {}, new TEvents::TEvWakeup));
```

Comment:
Why the extra allocation? BTW, you can use ev->Cookie instead of Tag in scheduled message to distinguish between different sources.

Reply:
Got rid of the unnecessary allocations, utilized cookie instead